week3/word.cpp

```diff
@@ -0,0 +1,18 @@
+//
+// Created by julia on 6/8/17.
+//
+
+#include "word.h"
```

Comment:
I think the idea to have a Dictionary class with methods is_word / is_prefix is better, since you can choose/change the implementation freely later if necessity arises.
Now you have 2 sets (words and prefixes), but that's an implementation detail.
Maybe you later need to store a lot of words, so you can change to just one sorted vector, or a Trie.